supplementary/how_2_document.md

```diff
@@ -71,4 +71,4 @@ So what to do:
 
 ** // TODO: add component_template.md//**
 
-If there is no sub page for a certain hardware component (e.g. a Wifi + Bluetooth chip) you can also create a new hardware component sub page from the template. Just open up the _New-component-template.md_ file, save as _Your-component.md_, fill with info and submit to the docs team.
```

Comment:
I was thinking the same thing .... github web editor doing something weird? invisible EOL stuff?